apps/docs/docs/template/api/virtual-scrolling.md

```diff
@@ -690,10 +689,146 @@ import { RxVirtualScrollStrategy } from '@rx-angular/template/experimental/virtu
+| layout technique                        | absolutely position each view                                                                 | transform a container within the viewport               |
+| scheduling technique                    | [`RenderStrategies`](../../cdk/render-strategies/strategies/concurrent-strategies.md)         | `requestAnimationFrame`                                 |
+| renderCallback                          | ✅                                                                                            | ❌                                                      |
+| SSR                                     | ⚠ - to be tested                                                                              | ✅                                                      |
+| Define visible view buffer              | configurable amount of views displayed in scroll direction,<br/>and opposite scroll direction | configurable buffer in px                               |
+| trackBy                                 | ✅                                                                                            | ✅                                                      |
+| View recycling                          | ✅                                                                                            | ✅                                                      |
+| Support scrollToIndex                   | ✅                                                                                            | ✅                                                      |
+| FixedSizeStrategy                       | ✅                                                                                            | ✅                                                      |
+| AutosizeStrategy                        | ✅                                                                                            | ✅ - ⚠️ scrollToIndex & scrolledIndex are not supported |
+| DynamicSizeStrategy                     | ✅                                                                                            | ❌                                                      |
+| Viewport orientation                    | ❌ - planned                                                                                  | ✅                                                      |
+| Separate viewport and scrolling element | ❌ - planned                                                                                  | ✅                                                      |
+| Tombstone / placeholder views           | ❌ - planned                                                                                  | ❌                                                      |
+
+For more information about the planned features for this package, see the [roadmap section](#missing-features-roadmap).
```

Comment:
I would suggest rename this section "Further improvements" or "Missing features" instead of roadmap and opening issues that could maybe encourage people to contribute.